src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureForOther {
+    public IsoscelesTrapezoid(String color, int a, int b, int c) {
+        super(color, a, b, c);
+    }
+
+    @Override
+    public double calculatorArea() {
+        return Math.pow(getThirdSide(), 2) - Math.pow(getSecondSide() - getFirstSide(), 2);
```

Comment:
area formula is invalid. correct formula will sound like "height * (top + bottom) / 2"